mod_sample/controllers.py

```diff
@@ -42,7 +42,7 @@ class SampleNotFoundException(Exception):
 
     def __init__(self, message):
         Exception.__init__(self)
-        self.message = message
+        self.= message
```

Comment:
This won't work. Did you use an IDE for this?

Reply:
Typo